{
    "id": 186969,
    "name": "CVE-2022-28958",
    "cwe": "Unknown",
    "modified": "2023-11-29T20:15:07",
    "published": "2022-05-18T10:15:08",
    "status": "Rejected",
    "summary": "Rejected reason: DO NOT USE THIS CVE RECORD. ConsultIDs: none. Reason: This record was withdrawn by its CNA. Further investigation showed that it was not a security issue. Notes: none.",
    "cvss3": null,
    "cvss2": null,
    "cpes": []
}